File: app/src/main/java/com/github/alexeygorovoy/moxytemplate/dagger/demo/splash/SplashComponent.java
package com.github.alexeygorovoy.moxytemplate.dagger.demo.splash;

import com.github.alexeygorovoy.moxytemplate.ui.demo.splash.view.SplashFragment;

import dagger.Subcomponent;


@Subcomponent(modules = {SplashModule.class})
public interface SplashComponent {
    void inject(SplashFragment fragment);
}

File: app/src/main/java/com/github/alexeygorovoy/moxytemplate/dagger/demo/splash/SplashComponent.kt
package com.github.alexeygorovoy.moxytemplate.dagger.demo.splash

import com.github.alexeygorovoy.moxytemplate.ui.demo.splash.view.SplashFragment

import dagger.Subcomponent

@Subcomponent(modules = [SplashModule::class])
interface SplashComponent {
    fun inject(fragment: SplashFragment)
}
